Main.rb

```diff
@@ -0,0 +1,9 @@
+# Change filename to Main.rb
+require './CardValidator'
+puts "Enter card numbers: "
+while true
+    cardNumber = gets.chomp.delete(" ")
+    card = Card.new(cardNumber)
+    card.validate
+    puts "#{card.type}: #{cardNumber} #{card.valid}" 
```

Comment:
From a UI perspective, it would be good user feedback to add another message asking `Please input a number:` after each input.